js/infobar.js

```diff
@@ -0,0 +1,466 @@
+'use strict';
+
+var $infoBar = {};
+var $message = {};
+var $acceptButton = {};
+var $cancelButton = {};
+var bars = {};
+var opened = false;
+
+var bar = {};
+var json = {};
+var lang = '';
+var availableLangs = {};
+var acceptLangs = [];
```

Comment:
One other nicety for maintenance (and review) would be commenting at least some of these config vars. For example, what's the difference between `$infoBar` and `bar`? Where do `availableLangs` come from?